model/gen-ai/spans.yaml

```diff
@@ -97,14 +97,35 @@ groups:
       This span represents a client call to Generative AI model or service that generates
       a response or requests a tool call based on the input prompt.
     note: |
+      This span is to be used when the models is running in a different process or when
+      the GenAI call happens over instrumented protocol such as HTTP.
+      In other cases, [Embedded Inference](/docs/gen-ai/gen-ai-spans.md#embedded-inference) span should be used.
+
       **Span name** SHOULD be `{gen_ai.operation.name} {gen_ai.request.model}`.
       Semantic conventions for individual GenAI systems and frameworks MAY specify different span name format
       and MUST follow the overall [guidelines for span names](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.37.0/specification/trace/api.md#span).
+    extends: attributes.gen_ai.inference.client
+    attributes:
+      - ref: gen_ai.system
```

Comment:
`gen_ai.system` now renamed to `gen_ai.provider.name` at https://github.com/open-telemetry/semantic-conventions/pull/2046

Also I can see the only difference of `span.gen_ai.embedded.internal` and `span.gen_ai.inference.client` is one is using `gen_ai.system` which has been deprecated and another is using `gen_ai.provider.name`, what is the use case here? Thanks